packages/ag-charts-enterprise/src/features/error-bar/errorBarNode.ts

```diff
@@ -85,5 +95,49 @@ export class ErrorBarNode extends _Scene.Group {
         }
         caps.path.closePath();
         caps.markDirtyTransform();
+
+        const { components } = this.bboxes;
+        components.length = 0;
+        if (yBar !== undefined) {
+            const whiskerHeight = yBar.lowerPoint.y - yBar.upperPoint.y;
+            components.push(
+                new _Scene.BBox(yBar.lowerPoint.x, yBar.upperPoint.y, whisker.strokeWidth, whiskerHeight),
+                new _Scene.BBox(yBar.lowerPoint.x - capOffset, yBar.lowerPoint.y, capLength, caps.strokeWidth),
+                new _Scene.BBox(yBar.upperPoint.x - capOffset, yBar.upperPoint.y, capLength, caps.strokeWidth)
+            );
+        }
+        if (xBar !== undefined) {
+            const whiskerWidth = xBar.upperPoint.x - xBar.lowerPoint.x;
+            components.push(
+                new _Scene.BBox(xBar.lowerPoint.x, xBar.upperPoint.y, whiskerWidth, whisker.strokeWidth),
+                new _Scene.BBox(xBar.lowerPoint.x, xBar.lowerPoint.y - capOffset, caps.strokeWidth, capLength),
+                new _Scene.BBox(xBar.upperPoint.x, xBar.upperPoint.y - capOffset, caps.strokeWidth, capLength)
+            );
+        }
+        const expansion = typeof range === 'string' ? 0 : range;
+        if (expansion > 0) {
+            for (const bbox of components) {
+                bbox.grow({ top: expansion, bottom: expansion, left: expansion, right: expansion });
+            }
+        }
+        this.bboxes.union = _Scene.BBox.merge(components);
+    }
+
+    override pickNode(x: number, y: number): _Scene.Node | undefined {
+        const point = this.transformPoint(x, y);
+        if (this.containsPoint(point.x, point.y)) {
+            return this;
+        }
+    }
+
+    override containsPoint(x: number, y: number): boolean {
+        if (this.bboxes.union.containsPoint(x, y)) {
+            for (const bbox of this.bboxes.components) {
+                if (bbox.containsPoint(x, y)) {
+                    return true;
+                }
+            }
+        }
+        return false;
```

Comment:
Nit: Could we reduce the nesting here by short-circuiting the outer condition?

e.g.:
```
if (!this.bboxes.union.containsPoint(x, y)) return false;
```

Reply:
Sure, happy to make this change. I generally believe that having few `return` statements makes the code more readable, but I don't think that it matters much in this case.

Reply:
Done